src/CreatePInvokeTxtFiles.ps1

```diff
@@ -0,0 +1,38 @@
+<#
+.SYNOPSIS
+Exports all the static methods names (independent of their visibility) that have [DllImport] attributes on them.
+If the DllImport.EntryPoint property is found that will be the value exported otherwise the method name.
+.DESCRIPTION
+TODO
+.PARAMETER AssemblyPath
+The path, including the file name, of the PInvoke lib from which the methods name will be exported.
+#>
+Param(
+    [Parameter(Mandatory=$true)]
+    [string]$AssemblyPath
+)
+$sb = New-Object -TypeName "System.Text.StringBuilder"
```

Comment:
Never mind, just realized that the EntryPoint property value could be different to the Name of the methods. I will fix it.